### PR TITLE
Switch between using CODE and DEV environments locally.

### PR DESCRIPTION
**Pull request by susiecoleman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,16 @@ You'll need the [AWS CLI](http://docs.aws.amazon.com/cli/latest/userguide/instal
 for the composer AWS account from [janus](https://janus.gutools.co.uk). You'll also need to follow the
 'Install SSL certificates' step in the [dev-nginx readme](https://github.com/guardian/dev-nginx). Then:
 
- - Fetch config from S3: `./fetch-config.sh`
+### Local Config
+
+This project uses [Configuration Magic](https://github.com/guardian/configuration-magic/) so you need to fetch a config file to use locally. There are 2 options and 2 scripts:
+ 
+ 1. `./fetch-DEV-config.sh` This uses the DEV database and DEV kinesis stream
+ 2. `.fetch-CODE-config.sh` This uses the CODE database and CODE kinesis stream.
```
Comment (marialivia16) [inline README.md]: You have a missing `/`

File: fetch-CODE-config.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```
Comment (marialivia16) [inline fetch-CODE-config.sh]: These 2 scrips could be merged into one and use parameters to set the stage.

File: README.md (hunk 1 above)
Comment (susiecoleman) [inline README.md]: Added

File: fetch-CODE-config.sh (hunk 2 above)
Comment (susiecoleman) [inline fetch-CODE-config.sh]: But this was so much less effort... Fixed it to use params 😃

**Pull request merged**